### ISSUE TITLE
dependency isn't represented when parent found but item missing

### ISSUE THREAD

**Issue by grahamdyson** [DevSnicket/eunice#51]
The following dependency wont be represented because the dependency's item is missing.

```yaml
- id: dependent
  dependsUpon: { id: parent, items: missing child }
- id: parent
```

Dependencies such as these could still be represented, but as being on the parent, which can be found.

**Comment by grahamdyson** [comment]
released in [JavaScript 6.19.0](https://devsnicket.com/eunice/javascript/releases#6.19.0)

**Comment by grahamdyson** [comment]
released in [JavaScript 6.23.0](https://devsnicket.com/eunice/javascript/releases#6.23.0)
